### PR TITLE
Hidden commands are available in technical formats

### LINKED ISSUE #7335: Hidden console commands are not hidden in JSON and XML descriptors
Once this PR is merged (https://github.com/symfony/symfony-docs/pull/7052) we need to update it to add a note about hidden commands not being hidden in JSON and XML descriptors.

See https://github.com/symfony/symfony/pull/21075

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Hello,

this fix https://github.com/symfony/symfony-docs/issues/7335.

We should rebase 3.2 docs into master first, as this pull request should be merged in master but the file I edit is not available, yet :)

Regards,

### CODE REVIEW

**Comment by xabbuh** [comment]
@mickaelandrieu I have merged branches up. You can now rebase and change the target branch. :)

**Comment by mickaelandrieu** [comment]
@xabbuh should be ok right now 👍

**Review by javiereguiluz — APPROVED** [review]
👍

File: console/hide_commands.rst [hunk 1]
```diff
@@ -30,3 +30,8 @@ In those cases, you can define the command as **hidden** by setting the
 
 Hidden commands behave the same as normal commands but they are no longer displayed
 in command listings, so end-users are not aware of their existence.
+
+.. note::
+
+    Hidden commands are still available when using the JSON or XML descriptors.
```
Comment (xabbuh) [inline console/hide_commands.rst]: descriptor

Comment (mickaelandrieu) [inline console/hide_commands.rst]: done

**Comment by xabbuh** [comment]
Thank you Mickaël.

**Pull request merged**